telemetry-testing/build.gradle.kts

```diff
@@ -52,5 +52,8 @@ tasks.test {
     // otel.exporter.otlp.protocol - the transport protocol to use on OTLP trace, metric, and log requests
     // otel.metric.export.interval - the interval, in milliseconds, between the start of two export attempts
     // More information: https://github.com/open-telemetry/opentelemetry-java/blob/main/sdk-extensions/autoconfigure/README.md
-    jvmArgs = listOf("-javaagent:${agent.singleFile}", "-Dotel.exporter.otlp.protocol=http/protobuf", "-Dotel.metric.export.interval=5000")
+    jvmArgs = listOf("-Xmx1G", "-javaagent:${agent.singleFile}",
+        "-Dotel.exporter.otlp.endpoint=http://127.0.0.1:4318",
+        "-Dotel.exporter.otlp.protocol=http/protobuf",
```

Comment:
are these the defaults now in 2.0.0? or did we not update the default endpoint when we updated the default protocol?

Reply:
I was having problems locally where the build failures looked like this:

```
Failed to export metrics. The request could not be executed. Full error message: Failed to connect to localhost/0:0:0:0:0:0:0:1:4318
```

I had hoped that standardizing everything to ipv4 would help, and it does when running locally on MacOS Sonoma. The `http/protobuf` was just to be sure, but I think it isn't needed.